modopt/base/backend.py

```diff
@@ -60,7 +60,8 @@ def get_backend(backend):
 
     Returns
     -------
-    The module for carrying out calculations
+    module
```

Comment:
`return LIBRARIES[backend], backend` looks like a `tuple` to me.

```suggestion
    tuple
```

Reply:
Ahhh, my bad, fixing the documentation